feedstock/requirements.txt

```
@@ -1 +1,3 @@
 git+https://github.com/pangeo-forge/pangeo-forge-recipes.git@beam-refactor
```

```suggestion
git+https://github.com/norlandrhagen/pangeo-forge-recipes.git@testing/combinefragments
```
@jbusecke I think accepting this comment will get us past the error you reported in https://github.com/leap-stc/data-management/pull/21#issuecomment-1577512896.

This is the branch for https://github.com/pangeo-forge/pangeo-forge-recipes/pull/521. Once that PR goes in, we can revert this change, but testing it from the PR branch would be useful as a gut check on the fix there.